README.md

```diff
@@ -92,7 +92,7 @@ You can directly copy and paste all the commands from the text given here into t
 
     CREATE TABLE Seller_Phone_num
     (
-        Phone_num NUMBER(10) NOT NULL,
+        Phone_num int NOT NULL,
```

Comment:
A phone number is of 10 digits, NUMBER(10) enforces that constraint. If changes to int, we would also have to add a constraint in schema to make sure that length of `Phone_num` is 10.